ome_zarr/reader.py

```diff
@@ -541,7 +541,8 @@ def get_tile(tile_name: str) -> np.ndarray:
             LOGGER.debug("LOADING tile... %s with shape: %s", path, tile_shape)
 
             try:
-                data = self.zarr.load(path)
+                # compute() to get the data from dask - we want to return array
+                data = self.zarr.load(path).compute()
```

Comment:
Should the computation not happen as late as possible, e.g., during the concatenation below, to make use of dask's scheduler?

Reply:
Aren't we already making use of dask's scheduler? Or do you mean make *better* use of it?

Would that actually give better performance? Not being very familiar with it, I'm reading https://docs.dask.org/en/stable/scheduler-overview.html which says there are gains to be made if you have a large graph and "share many intermediates", but I don't see that we can do that here?

I've simplified the code a fair bit in ee9ef7d to remove the extra layer of `delayed()` wrapping. This looks cleaner and still works as expected. It just means that we call `self.zarr.load(path)` for every tile right at the start, even if we never need to load/compute that region. I don't know if this makes any difference to performance but probably not?

Reply:
Better use, yes, so that the right things happen parallel.